CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-# Upcoming
+# 1.4.0
+
+## Features
+- **M3Query**: write endpoint support for M3-Map-Tags-JSON header in influxdb ([#3816](https://github.com/m3db/m3/pull/3816))
+- **M3Query**: Add support for `last_over_time` in m3query ([#3884](https://github.com/m3db/m3/pull/3884))
+- **M3Aggregator**: Add p75/p25 as aggregation options ([#3867](https://github.com/m3db/m3/pull/3867))
+
+## Bug Fixes
+- **M3DB**: Fix m3tsz encoding precision issue ([3872](https://github.com/m3db/m3/pull/3872))
```

Comment:
nit: m3tsz should be capitalized: M3TSZ

Reply:
Also can we change to “Fix M3TSZ to be deterministic when encoding high precision values”